src/components/ChatEntry.js

```diff
@@ -1,22 +1,41 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  let senderType = 'local';
+  if (props.sender !== 'Vladimir') {
```

Comment:
Tip: This is a little brittle since it means we can't re-use these components for a chat between different people.